[2021-02-06 06:42:45,834] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-02-05T00:00:00+00:00 [queued]>
[2021-02-06 06:42:45,846] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-02-05T00:00:00+00:00 [queued]>
[2021-02-06 06:42:45,846] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-06 06:42:45,846] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-06 06:42:45,846] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-06 06:42:45,855] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_2> on 2021-02-05T00:00:00+00:00
[2021-02-06 06:42:45,859] {standard_task_runner.py:52} INFO - Started process 87 to run task
[2021-02-06 06:42:45,861] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-02-05T00:00:00+00:00', '--job-id', '15', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpnilpr9p9', '--error-file', '/tmp/tmptot3an4k']
[2021-02-06 06:42:45,862] {standard_task_runner.py:77} INFO - Job 15: Subtask runme_2
[2021-02-06 06:42:45,899] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-02-05T00:00:00+00:00 [running]> on host 6be8c7a30c6f
[2021-02-06 06:42:45,933] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-02-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-05T00:00:00+00:00
[2021-02-06 06:42:45,933] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-02-06 06:42:45,933] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_2__20210205" && sleep 1
[2021-02-06 06:42:45,941] {bash.py:169} INFO - Output:
[2021-02-06 06:42:45,943] {bash.py:173} INFO - example_bash_operator__runme_2__20210205
[2021-02-06 06:42:46,944] {bash.py:177} INFO - Command exited with return code 0
[2021-02-06 06:42:46,967] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20210205T000000, start_date=20210206T064245, end_date=20210206T064246
[2021-02-06 06:42:46,992] {taskinstance.py:1219} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-06 06:42:47,035] {local_task_job.py:146} INFO - Task exited with return code 0
